pkg/operator/ceph/webhook.go

```diff
@@ -17,6 +17,8 @@ limitations under the License.
 package operator
 
 import (
+	"os"
```

Comment:
you should be able to revert this `os` import change that moved it

Reply:
Done.

Reply:
did the linter add this again? If you delete the blank line 21 it should fix this.